doc/packages.md

```diff
@@ -81,6 +81,7 @@
 | ooce/library/onig		| 6.9.5		| https://github.com/kkos/oniguruma/releases/ | [omniosorg](https://github.com/omniosorg)
 | ooce/library/pango		| 1.45.3	| https://download.gnome.org/sources/pango/cache.json https://ftp.gnome.org/pub/GNOME/sources/pango/ | [omniosorg](https://github.com/omniosorg)
 | ooce/library/protobuf		| 3.12.1	| https://github.com/protocolbuffers/protobuf/releases/ | [omniosorg](https://github.com/omniosorg)
+| ooce/library/serf		| 1.3.9 	| https://downloads.apache.org/ | [omniosorg](https://github.com/omniosorg)
```

Comment:
This URL is used by the update checker so can you point it straight at the serf folder?
```suggestion
| ooce/library/serf		| 1.3.9 	| https://downloads.apache.org/serf/ | [omniosorg](https://github.com/omniosorg)
```